health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/SidePanel.js

```
@@ -139,38 +175,22 @@
+          <div className={`slider-body ${sections && sections.length > 0 ? "with-sections" : ""}`}>
             {isOpen
               ? sections && sections?.length > 0
                 ? sections?.map((section, index) => (
                     <div className="section-divider-wrapper">
                       <div key={index} className="slider-section">
                         {section}
                       </div>
-                      {index < sections.length - 1 && (
-                        <div className="section-divider"></div>
-                      )}
+                      {index < sections.length - 1 && <div className="section-divider"></div>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)